conf/cassandra-dbaas-light.yaml

```diff
@@ -467,10 +467,14 @@ seed_provider:
 #
 # For materialized view writes, as there is a read involved, so this should
 # be limited by the less of concurrent reads or concurrent writes.
-concurrent_writes: 2
-concurrent_reads: 2
-concurrent_counter_writes: 2
-concurrent_materialized_view_writes: 2
+#
+# A value of 0 means direct execution.
```

Comment:
Nit: `A value of 0 means direct execution.` should be added to cassandra.yaml as well